core/drivers/ls_gpio.c

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
It would be good to have description for each function.
Static functions will have description in .c file and other function in .h